_posts/2016-10-18-using-rspecs-documentation-format.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title:  "Brief look at RSpec's formatting options"
+date: 2016-10-18 14:29:00
+categories: ["rspec", "ruby"]
+author: "mauro-oto"
+---
+
+A few weeks ago, I noticed weird output in the RSpec test suite (~4000 tests)
+for a Rails application:
+
+```
+................................................................................
+................................................................................
+................................................................................
+.............unknown OID 353414: failed to recognize type of '<field>'. It will
+be treated as String............................................................
+................................................................................
+```
+
+This Rails app uses a PostgreSQL database. After some Googling, it turns out
+that this is a warning from PostgreSQL. When the database doesn't recognize the
+type to use for a column, it casts to string by default.
```

Comment:
It's a Postgres thing. Good idea, I'll try to find something about it and add a source :+1: